### DISCUSSION TITLE
关于node的properties属性初始化不生效的问题

### DISCUSSION THREAD

**Discussion by kendrickcheung007** [didi/LogicFlow#1426] [General]
```javascript
lf.render({
      nodes: [
        {
          id: '1',
          type: 'rect',
          x: 100,
          y: 100,
          text: '一个正方形',
          properties: {
            backgroundColor: 'rgb(255, 230, 204)',
            borderWidth: '1px',
            borderColor: 'rgb(215, 155, 0)',
          },
        },
})
```

渲染完成后，没有properties中设置的效果

**Comment by wumail** [comment]
不知道你是期望直接通过在properties中设置样式并生效（这种方式显然不行），还是会在节点里更具properties处理样式，如果是后者，先确认节点Model中的properties是否被正常赋值

**Comment by kendrickcheung007** [comment]
> 不知道你是期望直接通过在properties中设置样式并生效（这种方式显然不行），还是会在节点里更具properties处理样式，如果是后者，先确认节点Model中的properties是否被正常赋值

嗯，前者确实不行，我现在改用文档那种继承基础node自定义样式的方法解决了。https://site.logic-flow.cn/docs/#/zh/guide/basic/node?id=%e8%87%aa%e5%ae%9a%e4%b9%89%e4%b8%80%e4%b8%aa%e4%b8%9a%e5%8a%a1%e8%8a%82%e7%82%b9
